packages/aws-cdk-lib/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -75,6 +75,14 @@ export enum NodegroupAmiType {
    * Amazon Linux 2023 (x86-64)
    */
   AL2023_X86_64_STANDARD = 'AL2023_x86_64_STANDARD',
+  /**
+   * Amazon Linux 2023 Neuron (x86-64)
+   */
+  AL2023_X86_64_NEURON = 'AL2023_x86_64_NEURON',
+  /**
+   * Amazon Linux 2023 with Nvidia-GPU support (x86-64)
+   */
+  AL2023_X86_64_NVIDIA = 'AL2023_x86_64_NVIDIA',
```

Comment:
See above

Reply:
Thank you so much. Comment updated. Im not really understand what these AMI has. Just want to fix the issue 😃